tests/unit/app/components/client_portal_modules.test.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import ClientPortalModules from 'app/js/components/client_portal_modules/client_portal_modules'
+
+describe('<ClientPortalModules />', () => {
+  it('renders with an empty list of modules', () => {
+    const modules = []
+    const component = shallow(<ClientPortalModules modules={modules} />)
+    expect(component.find('a').length).toBe(0)
+  })
+
+  it('renders with a single module', () => {
```

Comment:
do we need to check for a single module? I think only the second spec is enough.